docs/development/local_setup.md

```diff
@@ -320,36 +337,41 @@ $ vagrant destroy --force
 ==> core-01: Forcing shutdown of VM...
 ==> core-01: Destroying VM and associated drives...
 
-$ cd $GOPATH/src/github.com/gardener/gardener
+$ cd ..
 ```
 
-#### Start the Gardener Local Provider
-
-The Seed cluster provides the possibility to create Shoot clusters on several cloud provider. The Gardener Provider implements a common interface to all supported cloud providers. Here, the corresponding Gardener Provider for Local is used.
+#### Create, access and delete a Shoot Cluster
 
-By executing
+Now, you can create a Shoot cluster by running
 
 ```bash
-$ make start-local
-2018/02/14 10:53:34 Listening on :3777
-2018/02/14 10:53:34 Vagrant directory /Users/foo/go/src/github.com/gardener/gardener/vagrant
-2018/02/14 10:53:34 user-data path /Users/foo/git/go/src/github.com/gardener/gardener/dev/user-data
+$ kubectl apply -f dev/90-shoot-local.yaml
+shoot "local" created
 ```
 
-the Gardener Local Provider is started.
+Wait until the 2 secrets `osc-result-cloud-config-local-*` appear in the Shoot cluster namespace and then copy `cloud_config` from the secret `osc-result-cloud-config-local-xxxxx-downloader` to the local file `dev/user-data`:
 
-At this point three processes should run in an individual terminal, the Gardener API server, the Gardener controller manager and finally the Gardener Local Provider.
+```bash
+$ kubectl get secrets -n shoot--dev--local | grep osc-result-cloud-config-local
+osc-result-cloud-config-local-640f6-downloader   Opaque                                1      70s
+osc-result-cloud-config-local-640f6-original     Opaque                                1      68s
 
-#### Create, access and delete a Shoot Cluster
+$ kubectl get secrets osc-result-cloud-config-local-640f6-downloader -n shoot--dev--local -o jsonpath="{.data.cloud_config}" | base64 -d > dev/user-data
+```
 
-Now, you can create a Shoot cluster by running
+Manually start the Vagrant machine:
```

Comment:
If I understand correctly, with the proposed changes the machine is not longer started automatically when the shoot cluster is created?

Reply:
Yes. The old way this was done has become obsolete and had to be deleted. Doing it the right way would involve further changes which can be added later. IMO this should be done with a CMI plugin for the local provider.

Reply:
:+1:  I appoint myself for doing so :-)